package.json

```diff
@@ -9,8 +9,8 @@
     "export": "next export",
     "build-prod": "run-s clean build export",
     "clean": "rimraf .next .swc out coverage",
-    "lint": "next lint",
-    "format": "next lint --fix && prettier '**/*.{json,yaml}' --write --ignore-path .gitignore",
+    "lint": "next lint --cache --cache-location .eslintcache .",
```

Comment:
Why the `format` script doesn't use the flag `--cache-location`?

Another question, by default, Next.js lint already uses a cache: https://nextjs.org/docs/pages/building-your-application/configuring/eslint#caching. Is it really necessary to add the cache in this case?